File: micro-bit/js/1-2.js
let count = 0;
basic.showNumber(0);

input.onButtonPressed(Button.A, function button_a() {
    count++;
    basic.showNumber(count);
})

input.onButtonPressed(Button.B, function button_b() {
    count = 0;
    basic.showNumber(count);
})

File: micro-bit/python/1-2.py
count = 0
basic.show_number(0)

def button_a():
    global count
    count+=1
    basic.show_number(count)

def button_b():
    global count
    count = 0
    basic.show_number(count)

input.on_button_pressed(Button.A, button_a)
input.on_button_pressed(Button.B, button_b)